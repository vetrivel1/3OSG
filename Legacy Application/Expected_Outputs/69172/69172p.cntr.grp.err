Inspire Designer NetConsole 8.2 
"c:\program files\gmc\inspire designer\pnettcnetclient.exe"  -s 207.98.205.66 \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd -difDataInput1 \\207.98.205.10\users\public\69172p.cntr.grp -o FirstPass -nowarnings -useincluded *  -f \\207.98.205.10\users\public\69172p.cntr.grp.txt -la \\207.98.205.10\users\public\69172p.cntr.grp.err              
13.6.2025 15:53:15.212 open \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd;; -useincluded "*" -nowarnings
13.6.2025 15:53:15.212 ok;40375;13458
13.6.2025 15:53:15.353 run 13458 -difDataInput1 \\207.98.205.10\users\public\69172p.cntr.grp -o FirstPass -nowarnings -useincluded *  -f \\207.98.205.10\users\public\69172p.cntr.grp.txt -la \\207.98.205.10\users\public\69172p.cntr.grp.err              
13.6.2025 15:53:15.353 ok;40376
13.6.2025 15:53:15.431 close 13458
13.6.2025 15:53:15.431 ok;40377
13.6.2025 15:53:15.571 wfjrc 40376
13.6.2025 15:53:30.329 ok;finished;2
13.6.2025 15:53:30.469 wfjrc 40377
13.6.2025 15:53:30.469 ok;finished;0
13.6.2025 15:53:30.610 qjm 40376
13.6.2025 15:53:30.610 ok;8242
13.6.2025 15:53:30.610 Status message 0856:Opening workflow 13458 //207.98.205.65/programs/gjs_002503_0001_lpsbill.wfd.

Job Messages:
Warning 7883:The .WFD file has been created by a newer version 10.1.1.4, program version 8.2.44.0.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/arial.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/arialbd.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/ariali.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/arialbi.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/ariblk.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/calibri.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/calibrib.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/calibrii.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\207.98.205.65\programs\gjs_002503_0001_lpsbill.wfd. File C:/Windows/Fonts/wingding.ttf on disk and included file are different, using included file.
Warning 7883:The .WFD file has been created by a newer version 10.1.1.4, program version 8.2.44.0.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/arial.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/arialbd.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/ariali.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/arialbi.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/ariblk.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/arial.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/arialbd.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/ariali.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/arialbi.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1.cnpjax.corp\Programs\AddrBlock_container_DP.wfd. File C:/Windows/Fonts/ariblk.ttf on disk and included file are different, using included file.
Status message 0857:Opened workflow //207.98.205.65/programs/gjs_002503_0001_lpsbill.wfd.
Status message 0860:Running commandline -difDataInput1 \\207.98.205.10\users\public\69172p.cntr.grp -o FirstPass -nowarnings -useincluded *  -f \\207.98.205.10\users\public\69172p.cntr.grp.txt -la \\207.98.205.10\users\public\69172p.cntr.grp.err.
Status message 0884:Running module FirstPass.
Warning 7883:The .WFD file has been created by a newer version 10.1.1.4, program version 8.2.44.0.
Warning 7883:The .WFD file has been created by a newer version 10.1.1.2, program version 8.2.44.0.
Warning 7883:The .WFD file has been created by a newer version 10.1.1.4, program version 8.2.44.0.
Warning 7882:Workflow: \\jaxgmc1\programs\NCP_Dist_Print_Perf.wfd. File C:/Windows/Fonts/arial.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1\programs\NCP_Dist_Print_Perf.wfd. File C:/Windows/Fonts/arialbd.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1\programs\NCP_Dist_Print_Perf.wfd. File C:/Windows/Fonts/ariali.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1\programs\NCP_Dist_Print_Perf.wfd. File C:/Windows/Fonts/arialbi.ttf on disk and included file are different, using included file.
Warning 7882:Workflow: \\jaxgmc1\programs\NCP_Dist_Print_Perf.wfd. File C:/Windows/Fonts/ariblk.ttf on disk and included file are different, using included file.
Warning 0952:Two or more fields on same level have identical name. Module name: DataAdd1
Warning 0952:Two or more fields on same level have identical name. Module name: DataInput1
Warning 0952:Two or more fields on same level have identical name. Module name: DataTransformer10
Status message 0120:Data Input: Opening: \\207.98.205.205\User Files\0976multi.csv.
Status message 0596:ImpositionScript1: Imposition script preprocessing started.
Status message 0120:Data Input: Opening: \\207.98.205.10\users\programs\2503_image.csv. (Image table)
Status message 0120:Data Input: Opening: \\207.98.205.65\programs\0310_pppenalty.txt.
Status message 0120:Data Input: Opening: \\jaxgmc1.cnpjax.corp\programs\0976_lossmittable.csv. (Loss Mitt Table)
Status message 0120:Data Input: Opening: \\207.98.205.10\users\programs\2503_transtable.csv.
Status message 0596:TransactionsTable: Imposition script preprocessing started.
Status message 0596:ImpositionScript3: Imposition script preprocessing started.
Warning 7040:ImpositionScript3: Variable 'sec_zone' has not been used. (5,35)
Status message 0120:Data Input: Opening: \\207.98.205.10\users\public\69172p.cntr.grp. (DataInput1)
Status message 0120:Data Input: Opening: \\thehp\users\programs\2503supptable.txt. (DataInput2)
Status message 0597:ImpositionScript3: Imposition script preprocessing finished.
Status message 0597:TransactionsTable: Imposition script preprocessing finished.
Status message 0597:ImpositionScript1: Imposition script preprocessing finished.
Status message 0045:Data opened (duration 0:00:01.482, records 5).
Status message 0007:Preprocessing started.
Status message 0008:Preprocessing finished (duration 0:00:00.016, sheets 10, tempsize 10687).
Status message 0641:Job started.
Status message 0003:Job finished (duration 0:00:01.856, sheets 10, sheets/min 323.28).
