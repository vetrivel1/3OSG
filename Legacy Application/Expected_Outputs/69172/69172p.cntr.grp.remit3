00050000546242018803,0020061255,06/05/2025,     784.58,12345,2169172,1000,2503,,,,48277150299
00050000546242018804,0020061458,06/05/2025,    4653.56,12345,2169172,1000,2503,,,,48277150299
00050000546242018805,0020061530,06/05/2025,    3984.56,12345,2169172,1000,2503,,,,48277150299
00050000546242018806,0020061618,06/05/2025,    2880.88,12345,2169172,1000,2503,,,,48277150299
00050000546242018807,6500001175,06/05/2025,    2701.04,12345,2169172,1000,2503,,,,48277150299
