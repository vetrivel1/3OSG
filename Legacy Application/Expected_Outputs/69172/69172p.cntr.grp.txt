000001|0020061255          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|1
000001|0020061255          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|0
000002|0020061458          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|1
000002|0020061458          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|0
000003|0020061530          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|1
000003|0020061530          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|0
000004|0020061618          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|1
000004|0020061618          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|0
000005|6500001175          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|1
000005|6500001175          |0002|000| | | | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |69172p.cntr.grp               |00001|00002|0
